packages/video_player/video_player_avfoundation/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.6.3
+
+* Supports the audio-only HLS (.m3u8) on iOS.
```

Comment:
```suggestion
* Bugfix to allow audio-only HLS (.m3u8) on iOS.
```

Reply:
@cbracken 
addressed this in the following commit!
[Tweak the CHANGELOG](https://github.com/flutter/packages/pull/7890/commits/041189a82881fdb78fffb249a7da47d1ae3c78b9)